src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -51,12 +51,19 @@ public static String buildSearchConceptsQuery(
     var domainClause =
         createDomainClause(conceptTablePointer, conceptTableVariable, domainOption.getCategory());
 
+    List<FieldVariable> select = List.of(nameField, idField, personIdField);
+
+    List<TableVariable> tables = List.of(conceptTableVariable, domainOccurenceTableVariable);
+
+    List<OrderByVariable> orderBy =
+        List.of(new OrderByVariable(personIdField, OrderByDirection.DESCENDING));
+
+    List<FieldVariable> groupBy = List.of(nameField, idField);
+
     // if the search test is empty do not include the search clauses
     // return all concepts in the specified domain
     if (searchText == null || searchText.isEmpty()) {
-      // TODO: DC-845 Implement pagination, remove hardcoded limit
```

Comment:
Do we want to keep this TODO item? If not here, maybe before line 96, or both places, I don't know, what do you think?

Reply:
I think it's best if we delete this because it's easier to track the ticket if it's in Jira. Does that sound ok?